torch/_dynamo/allowed_functions.py

```diff
@@ -152,6 +152,7 @@ def _is_allowed_module_prefix(obj):
         # these functions, rather than keep them opaque-ly in the graph.
         disallowed_modules = (
             "torch.optim.",
+            "torch.utils._foreach_utils",  # omit the period so we match all the functions in this module
```

Comment:
I'm curious if we can disallow the whole ```torch.utils.```.

Reply:
I'll try this out

Reply:
Actually this doesn't mean anything - you also need to add each file to the allowlist to enable it to be inlined